it/_contributing.md

```diff
@@ -0,0 +1,13 @@
+
+# Contributi
+
+"Linee guida di Sass" è un progetto libero di cui mi prendo cura nel mio tempo libero. Inutile a dirsi, c'è un bel po' di lavoro da fare per tenere tutto aggiornato, documentato e rilevante. 
```

Comment:
Even if "linee guida di Sass" is a matching translations for Sass guidelines, makes sense keeping the original title here.
I will correct it here: https://github.com/HugoGiraudel/sass-guidelines/pull/239